### PR TITLE
fix(log): ensure that directory for request.log exists

### PR DESCRIPTION
**Pull request by tripodsan**

### CODE REVIEW

File: lib/app.js [hunk 1]
```diff
@@ -38,7 +38,9 @@ function getLogFile(options) {
 }
 
 function getLogStream(options) {
-  return fse.createWriteStream(getLogFile(options), { flags: 'a' });
+  const logFile = getLogFile(options);
+  fse.ensureDirSync(path.dirname(logFile));
```
Comment (stefan-guggisberg) [inline lib/app.js]: Why is this necessary?  There's an `await fse.ensureDir(config.logs.logsDir);` in the `initConfiguration` (`server.js`, line 88).

Comment (tripodsan) [reply lib/app.js]: yes, but if there is already a logger passed via config, the logs directory might not exist (yet) and then it will not be initialized.

**Comment by tripodsan** [comment]
let's close it for now....
